tools/clang/tools/dxcompiler/dxcompilerobj.cpp

```diff
@@ -748,6 +749,7 @@ class DxcCompiler : public IDxcCompiler2,
       ::llvm::sys::fs::MSFileSystem *msfPtr;
       IFT(CreateMSFileSystemForDisk(&msfPtr));
       std::unique_ptr<::llvm::sys::fs::MSFileSystem> msf(msfPtr);
+
```

Comment:
This file is all whitespace changes. I'm pretty indifferent to whitespace changes, but unless I'm mistaken the blanks being added were just removed. If there's reason for this, I'm already on board whatever it is. I just want to verify it's not included by mistake.

Reply:
yeah, it's intentional. I'm bringing down the diff between the MS and google TOT.
We are down to only a few hundred lines of difference. Tempting to post all as one CL :laughing:  (But i won't of course). I'll be sending a few CLs to MS repo today to further bring down the difference. Hoping to reach zero difference soon (depending on how fast our PRs are reviewed).